clarify opacity < 1 creates stacking context (#42999)
mdn/content: files/en-us/web/css/guides/positioned_layout/stacking_context/index.md

</article>
```

> Every container element has an {{cssxref("opacity")}} of less than `1`. This creates a stacking context. The containers each also have a {{cssxref("position")}} of either `relative` or `absolute` set, creating a stacking context when the element has `z-index` value other than `auto`.

Reviewer: ```suggestion
Every container element has an {{cssxref("opacity")}} of less than `1` (which creates a stacking context), and a {{cssxref("position")}} of either `relative` or `absolute` (which creates a stacking context when the element also has a `z-index` value other than `auto`).
```

Maybe something like this? I still felt like the phrasing was a little confusing.
